src/Product.php

```diff
@@ -8,41 +8,57 @@
 class Product {
 
   /**
+   * The product name.
+   *
    * @var string
    */
   private $name;
 
   /**
+   * Unique identifier.
+   *
    * @var string
    */
   private $id;
 
   /**
+   * The price.
+   *
    * @var string
    */
   private $price;
 
   /**
+   * The brand.
+   *
    * @var string
    */
   private $brand;
 
   /**
+   * The category.
+   *
    * @var string
    */
   private $category;
 
   /**
+   * The product variation title.
```

Comment:
I will change this for a more generic comment.
Also, I think the mapping is pretty net now & people will be able to adapte it according their needs via the event so for me it's okay - nop ?

Reply:
Yep! I also think it would be hard to offer a configuration GUI for the mapping since the possibilities on how to setup the products are endlessly. Thinking about the field mapping - I think this functionality should be extracted to a service. :)

Reply:
I open an Issue to discuss this there & how to implement it.
https://github.com/gridonic/commerce_google_tag_manager/issues/15